test/testsuites/const.go

```diff
@@ -8,4 +8,7 @@ const (
 
 	// targetNamespace is the targetNamespace value of the CatalogSourceConfig used in this test suite.
 	targetNamespace string = "target-namespace"
+
+	// packageName is the non-existing package
+	packageName string = "non-existing-package"
```

Comment:
This variable name is misleading. I would assume `packageName` is a working package name. I suggest changing it to `nonExistingPackageName` for clarity.